robottelo/ui/computeresource.py

```diff
@@ -75,34 +119,29 @@ def search(self, name):
         element = self.search_entity(name, locators["resource.select_name"])
         return element
 
-    def update(self, oldname, newname, orgs, new_orgs, org_select=False,
-               provider_type=None, url=None, user=None, password=None,
-               region=None, libvirt_display=None, libvirt_set_passwd=True,
-               tenant=None, new_description=None):
-        """Updates a compute resource."""
-        element = self.search(oldname)
+    def update(self, name, newname=None, parameter_list=[],
+               orgs=None, org_select=None, locations=None, loc_select=None):
+        """Updates compute resource entity"""
+        element = self.search(name)
```

Comment:
Not sure that I understand point here as we should call ```search``` method in 100% of cases to be able to update necessary entity and we have to call an exception if entity for update was not found.